entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity.proto

```diff
@@ -42,11 +42,14 @@ message Entity {
   string version = 3;
   string entity_type = 4;
   string entity_name = 5;
-  //attributes that uniquely identify the entity
+  // attributes that uniquely identify the entity
   map<string, AttributeValue> identifying_attributes = 6;
 
-  //attributes that describe the entity (e.g. service labels, service version, lang etc)
+  // attributes that describe the entity (e.g. service labels, service version, lang etc)
   map<string, AttributeValue> attributes = 7;
+  // no need to set it. document-store automatically adds a `createdTime` field
+  // can directly read the value from the returned entity
+  int64 created_time = 8;
```

Comment:
please include unit
```suggestion
  int64 created_time_ms = 8;
```

Reply:
discussed offline - handcuffed here since this is an existing field in the doc already. Disregard.